service/topology_coordinator.cc

```diff
@@ -2295,6 +2295,10 @@ class topology_coordinator : public endpoint_lifecycle_subscriber {
                     muts.emplace_back(rtbuilder.build());
                     co_await update_topology_state(take_guard(std::move(node)), std::move(muts),
                                                    "bootstrap: read fence completed");
+                    // Make sure the load balancer knows the capacity for the new node immediately.
+                    (void)_tablet_load_stats_refresh.trigger().handle_exception([] (auto ep) {
+                        rtlogger.warn("Error during tablet load stats refresh: {}", ep);
```

Comment:
Ignoring a future needs a comment explaining why it is safe. Is it in fact safe?

Better, signal a fiber to do this (out of scope).

Reply:
> Ignoring a future needs a comment explaining why it is safe. Is it in fact safe?
> 
> Better, signal a fiber to do this (out of scope).

It's safe, we join() with _tablet_load_stats_refresh in topology_coordinator::close(), which waits for all triggered jobs.